jaxrs-tck/pom.xml

```diff
@@ -0,0 +1,100 @@
+
+    This Source Code may also be made available under the following Secondary
+    Licenses when the conditions for such availability set forth in the
+    Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+    version 2 with the GNU Classpath Exception, which is available at
+    https://www.gnu.org/software/classpath/license.html.
+
+    SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>jakarta.ws.rs</groupId>
+    <artifactId>jakarta.ws.rs-tck</artifactId>
+    <version>3.1-SNAPSHOT</version>
+
+    <name>Jakarta REST TCK</name>
+    <description>Technology Compatibility Kit for Jakarta RESTful Web Services</description>
+    <url>https://github.com/eclipse-ee4j/jaxrs-api</url>
+
+    <properties>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+    </properties>
+
+    <organization>
+        <name>Eclipse Foundation</name>
+        <url>https://www.eclipse.org/org/foundation/</url>
+    </organization>
+
+    <developers>
+        <developer>
+            <id>developers</id>
+            <name>JAX-RS API Developers</name>
+            <email>jaxrs-dev@eclipse.org</email>
+            <url>https://github.com/eclipse-ee4j/jaxrs-api/graphs/contributors</url>
+        </developer>
+    </developers>
+
+    <issueManagement>
+        <system>Github</system>
+        <url>https://github.com/eclipse-ee4j/jaxrs-api/issues</url>
+    </issueManagement>
+
+    <mailingLists>
+        <mailingList>
+            <name>JAX-RS Developer Discussions</name>
+            <archive>jaxrs-dev@eclipse.org</archive>
+        </mailingList>
+    </mailingLists>
+
+    <licenses>
+        <license>
+            <name>EPL-2.0</name>
+            <url>http://www.eclipse.org/legal/epl-2.0</url>
+            <distribution>repo</distribution>
+        </license>
+        <license>
+            <name>GPL-2.0-with-classpath-exception</name>
+            <url>https://www.gnu.org/software/classpath/license.html</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <scm>
+        <connection>scm:git:https://github.com/eclipse-ee4j/jaxrs-api</connection>
+        <url>https://github.com/eclipse-ee4j/jaxrs-api</url>
+        <tag>HEAD</tag>
+    </scm>
+
+    <dependencies>
+        <dependency>
+            <groupId>jakarta.ws.rs</groupId>
+            <artifactId>jakarta.ws.rs-api</artifactId>
+            <version>3.1-SNAPSHOT</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <version>[5.5.2, 5.6-A00)</version>
```

Comment:
For these types of dependencies, the TCK bundle should file CQs, I believe. But the CQ can only be filed for a certain version, not for a range.

Reply:
Just wondering, shouldn't we create a tool that extracts the dependencies from Maven and automatically files CQs?

In the case of a range, the tool could check Maven central, check for every available version in that range, and file CQs for each and every one fo them?

Reply:
@jansupol AFAIK ranges are allowed as long as the changes cover only bug fixes, but I will ask the EMO about it.

@arjantijms Sounds cool - do you have such a tool? Even better: This should be a bot on Github! ;-)

Reply:
I resolved the following answer from EF's @waynebeaton:
>Hi Markus.
>
>CQs are not required for service releases (bug fixes only) based on major or minor versions of content that has already been approved. 
>
>I don't tend to see too many version ranges in Maven dependencies, but there's nothing stopping you from using them (except, perhaps, the ire of build purists). The way that I tend to deal with dependencies when doing an IP Log check is to check project builds to see that what actually gets resolved by Maven has been vetted by the process.
>
>We're taking steps to provide a tool that provide some additional automation of the vetting process. The Dash License Tool is pushing in that direction. There's a pull request to turn it into a Maven plug-in that I'll accept shortly. You'll also notice some experimenting with review request automation with the "-review" switch.

Reply:
CQs have been approved by PMC meanwhile. :-)